src/main/scala/mesosphere/marathon/tasks/TaskTracker.scala

```diff
@@ -289,12 +304,13 @@ class TaskTracker @Inject() (
 object TaskTracker {
 
   private[marathon] class InternalApp(
-      val appName: PathId,
-      var tasks: TrieMap[String, MarathonTask],
-      var shutdown: Boolean) {
+                                       val appName: PathId,
```

Comment:
These instances of reformatting are somewhat strange. Did you do that on purpose?